gap/riscv64.patch

```diff
@@ -0,0 +1,18 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -51,6 +51,15 @@ prepare() {
+   cd CddInterface-*
+   sed -e 's|/usr/include/cdd|/usr/include/cddlib|' -i configure.ac # Fix build with cddlib 0.94k
+   rm configure
++  cd ..
++
++# Update the config.guess file
```

Comment:
Consider adding more indentation to this line, so that the # character can be vertically aligned with its sibling lines.